ida/data.yml

```diff
@@ -932,11 +932,11 @@ classes:
       - ea: 0x14222BD40
         pointer: True
     funcs:
-      0x1412EC2D0: HasValue  # g_FateTablePtr != 0
-      0x1412EC2F0: GetSingleton
-      0x1412EC300: Initialize
-      0x1412ECAD0: ctor
-      0x1412EC350: dtor
+      0x1412EC2D0: HasInstance
+      0x1412EC2F0: GetInstance
+      0x1412EC300: Create
+      0x1412EC350: Destroy
+      0x1412ECAD0: Initialize
```

Comment:
Should still be named `ctor`

Reply:
As we talked about it, the ctor allocates the thing. This address does not do that.